src/callbacks.cpp

```diff
@@ -158,15 +158,40 @@ cb_about (GtkAction *action, gpointer data)
     ProcData *procdata = static_cast<ProcData*>(data);
 
     const gchar * const authors[] = {
-        "Kevin Vandersloot",
+        "Adam Erdman <hekel@archlinux.info>",
+        "Alexander Pyhalov <apyhalov@gmail.com>",
+        "Andreas Nilsson <nisses.mail@home.se>",
+        "Benoît Dejean <bdejean@gmail.com>",
+        "Chris Kühl <chrisk@openismus.com>",
+        "Clement Lefebvre <clement.lefebvre@linuxmint.com>",
+        "Elias Aebi <user142@hotmail.com>",
         "Erik Johnsson",
         "Jorgen Scheibengruber",
-        "Benoît Dejean",
-        "Paolo Borelli",
         "Karl Lattimer",
-        "Perberos",
-        "Stefano Karapetsas",
-        "Steve Zesch",
+        "Kevin Vandersloot",
+        "Laurent Napias <tamplan@free.fr>",
+        "Marcel Dijkstra <marcel.dykstra@gmail.com>",
+        "Martin Wimpress <martin@mate-desktop.org>",
+        "Matias De lellis <mati86dl@gmail.com>",
+        "Mike Gabriel <mike.gabriel@das-netzwerkteam.de>",
+        "Nelson Marques <nmo.marques@gmail.com>",
+        "Obatta Akio <obache@users.noreply.github.com>",
+        "Pablo Barciela <scow@riseup.net>",
+        "Paolo Borelli",
+        "Perberos <perberos@gmail.com>",
+        "Piotr Drąg <piotrdrag@gmail.com>",
+        "Robert Buj <robert.buj@gmail.com>",
+        "Sander Sweers <infirit@gmail.com>",
+        "Scott Balneaves <sbalneav@alburg.net>",
+        "Soong Noonien <49582808+SoongNoonien@users.noreply.github.com>",
```

Comment:
`@users.noreply.github.com`

this is a valid mail?

Reply:
Yes, it's valid. The author checked the option **Keep my email address private** on GitHub account settings.

https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address